kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -73,12 +98,57 @@
   .title
     display: inline
 
+  .no-border
+    border: none
+
   .confirm-btn
     float: right
 
   .open-btn
     background-color: $core-bg-light
 
+  .user-field
+    padding-bottom: 5%
+    input
+      width: 100%
+      height: 40px
+      font-weight: bold
+      border: none
+      border-bottom: 1px solid #3a3a3a
+    label
+      position: relative
+    select
+      -webkit-appearance: menulist-button
+      width: 100%
+      height: 40px
+      font-weight: bold
+      background-color: transparent
+
+  .edit-form
+    width: 200px
+    margin: 0 auto
+    display: block
+    padding: 5px 10px
+    letter-spacing: 0.08em
+    border: none
+    border-bottom: 1px solid $core-text-default
+    height: 30px
+    &:focus
+      outline: none
+      border-bottom: 3px solid $core-action-normal
+
+  .edit-username
```

Comment:
@rayykim any reason we've transitioned to using backgrounds rather than doing the `<svg>` tags in the html like we were doing before?